docs/modules/operation/pages/search/search.adoc

```diff
@@ -0,0 +1,99 @@
+[[search-functionality]]
+= Search
+
+{page-component-title} provides two ways to search in the web UI: through the search bar and the *Search* view.
+
+== Search bar
+
+The search bar uses keywords to return both nodes that match the keyword(s) and a list of actions you can perform that relate to the keyword.
+Press the [shift] key twice or click in the search bar to activate it for typing.
```

Comment:
We can use the Keyboard macro `btn:[Shift]`.

Reply:
I learn something new everyday! Thanks updated.